Placeholders (#343)
commerce-docs/microsite-commerce-storefront: src/content/docs/merchants/get-started/multistore.mdx

**Select a store:**

- United States (USD)
  - [United States (USD)](https://main--aem-boilerplate-commerce--hlxsites.aem.page/en/#nolocal)
- Canada (CAD)
  - [Canada (EN)](https://main--aem-boilerplate-commerce--hlxsites.aem.page/en_ca/#nolocal)
```

> **Important**: The storefront will automatically update links to be relative to the root path of the storefront. For links you don't want to be localized (such as store switcher links), add the hash `#nolocal` to ensure these remain absolute to the site. This is particularly useful for store switchers as you want to be able to navigate to different store paths within the same site.

Reviewer: ```suggestion
**Important:** The storefront will automatically update links relative to the storefront's root path. For links you don't want to be localized (such as a store switcher links), add the hash `#nolocal` to ensure these remain absolute to the site. This is particularly useful for store switchers as you want to navigate to different store paths within the same site.
```

### Placeholders

> The placeholders directory contains JSON files that define reusable variables for text and UI components displayed on your storefront. Each drop-in component has its own placeholders file (e.g., `cart.json`, `checkout.json`, `pdp.json`). These files replace variables in your content files with the appropriate text values.

Reviewer: ```suggestion
In the placeholders directory, you will find JSON files that define reusable variables for text and UI components. Each drop-in component has its own placeholders file— `cart.json`, `checkout.json`, `pdp.json`. These files replace variables in your content files with text values.
```

<Steps>
1. Translate or localize content files in the `/en_ca/` folders.

> 1. Ensure all labels in the placeholders JSON files are accurately translated into the desired language. Update each component's placeholders file (e.g., `cart.json`, `checkout.json`, `pdp.json`) as needed.

Reviewer: ```suggestion
1. Make sure labels in the placeholder JSON files are correctly translated. Update each component's placeholders file as needed—`cart.json`, `checkout.json`, `pdp.json`.
```
